backend-docker/main.go

```diff
@@ -1,58 +1,40 @@
 package main
 
 import (
-	"encoding/json"
 	"flag"
 	"log"
-	"os"
-
 	"github.com/EUDAT-GEF/GEF/backend-docker/dckr"
 	"github.com/EUDAT-GEF/GEF/backend-docker/server"
+	"github.com/EUDAT-GEF/GEF/backend-docker/config"
 )
 
-var configFilePath = "config.json"
-var config configuration
-
-type configuration struct {
-	Docker []dckr.Config
-	Server server.Config
-}
+var configFilePath = "config/config.json"
```

Comment:
I disagree with this change, the config file must be placed in the base directory, to be easily found when we run the compiled binary.